addons/mass_mailing/models/mailing.py

```diff
@@ -525,6 +527,51 @@ def _get_remaining_recipients(self):
         done_res_ids = [record['res_id'] for record in already_mailed]
         return [rid for rid in res_ids if rid not in done_res_ids]
 
+    def _send_statistics_mails(self):
+        """Send an email to the responsible of each finished mailing with the statistics."""
+        self.kpi_mail_required = False
+
+        for mailing in self:
+            user = mailing.user_id
+
+            link_trackers = self.env['link.tracker'].search(
+                [('mass_mailing_id', '=', mailing.id)]
+            ).sorted('count', reverse=True)
+
+            link_trackers_body = self.env['mail.render.mixin']._render_encapsulate(
+                'mass_mailing.mass_mailing_kpi_link_trackers',
+                add_context={
+                    'object': mailing,
+                    'link_trackers': link_trackers,
+                },
+            )
+
+            rendered_body = self.env['mail.render.mixin']._render_encapsulate(
```

Comment:
Do a simple _render_template_qweb with previously rendered content given in render context.

Isn't it possible to keep "object" in digest_mail_main, as there is a "name" field on mailing it should work ?

Reply:
@tde-banana-odoo The name get of the mailing add the date in it :/ (and also it's not just the name of the mailing, but the subject in parenthesis with a title before)